### DISCUSSION TITLE
having trouble running mathesar

### DISCUSSION THREAD

**Discussion by SEGRR** [mathesar-foundation/mathesar#2357] [Developer Q&A] [answered]
### HI myself Sufiyan. I am really instrested in contributing in this project  I am having some trouble running project 

- when I try to run following command in my ubuntu 

`docker-compose up`

- I get error somtehing like this 

`ERROR:` Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/`

plz help soving this issue!

**Comment by kgodey** [comment]
@SEGRR Please try upgrading your Docker compose version.

**Comment by silentninja** [comment, marked as answer]
Can you specify your 
1. Docker version - Found using the command `docker -v`
2. Docker-compose version - Found using the command `docker-compose -v`


We use docker-compose file format version `3.9` so your docker engine version should be at least `19.03.0+`  If it is lower than `19.03.0+`  please refer to https://docs.docker.com/desktop/install/ubuntu/ on how to upgrade the package

**Reply by SEGRR** [reply]
Thank you for suggestion. I tried running docker using WSL and it worked!
